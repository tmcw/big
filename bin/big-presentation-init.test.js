/* eslint-env jest */
var fs = require("fs");
var os = require("os");
var path = require("path");
var init = require("./big-presentation-init.js");

var workingDirectory;

beforeEach(function() {
  var tmp = path.join(os.tmpdir(), "./tmp-" + Date.now());
  fs.mkdirSync(tmp);
  workingDirectory = tmp;
});

test("init", function() {
  process.chdir(workingDirectory);
  init(Object.assign({}, process, { argv: [] }));
  expect(fs.readdirSync(workingDirectory)).toMatchSnapshot();
});
