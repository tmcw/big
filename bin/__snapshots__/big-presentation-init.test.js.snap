// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`init 1`] = `
Array [
  "big.css",
  "big.js",
  "highlight.css",
  "highlight.js",
  "index.html",
  "rubik-v4-latin-regular.woff",
  "rubik-v4-latin-regular.woff2",
]
`;
